src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -44,7 +47,10 @@
           "type": "keyword"
         },
         "uninstalled": {
-          "type": "keyword"
+          "type": "array",
+          "items": {
```

Comment:
What do you think about renaming `items` to `properties`?

Reply:
I tried to follow similarities to the JSON-schema specification (even though we are not fully using that).

I think `items` or an array make more sense to me. They are not `properties` or the array. 

But happy to change it if you think `properties` is the right way to go.